rest_framework_tus/views.py

```diff
@@ -145,6 +146,14 @@ def create(self, request, *args, **kwargs):
         # Validate headers
         headers = self.validate_success_headers(headers)
 
+        # Upload length is zero bytes, so PATCH will not be called.
+        # Do the needed stuff here:
```

Comment:
@wadevries commented on 2022-03-16 14:08:41

This comment can be improved; what is the ‘needed stuff’?